.github/workflows/check.yml

```diff
@@ -455,10 +455,10 @@ jobs:
       with:
         version: stable
         targets: ${{ matrix.target }}
-        tools: cargo-ndk
+        tools: cargo-ndk@^4
```

Comment:
How about pinning a concrete version here?

Reply:
We should do this in parallel with neqo. I wonder if it makes sense to factor out the Android (and VM) stuff into reusable actions in neqo and then use them here, like we do for the Rust action.